Makefile

```diff
@@ -15,8 +15,11 @@ test:
 	npm run test:coverage
 
 .PHONY: test-unit
-test-unit:
-	npm run test
+test-unit: clean
+	# 'ECS pipeline calls test-unit so need to run sonar-test here'
+	npm run sonar-test
+#test-unit:
```

Comment:
Commented out lines here can be deleted.

Reply:
Done